shenyu-examples/shenyu-examples-dubbo/shenyu-examples-spring-cloud-alibaba-dubbo-service-annotation/src/main/java/org/apache/shenyu/examples/spring/cloud/dubbo/service/annotation/impl/MultiParamServiceImpl.java

```diff
@@ -32,7 +32,7 @@
 /**
  * The type Dubbo multi param service.
  */
-@DubboService
+@ShenyuDubboService("demo")
```

Comment:
rollback this

Reply:
Why was this not rollbacked? It seems that Dubbo only resolves the direct annotation, not composite annotation.